### ISSUE TITLE
Standard command builtins are now enabled by default but shouldn't be

### ISSUE THREAD

**Issue by dannyweldon** [att/ast#380]
It has been mentioned several times that the `ksh` builtin replacements for the standard commands (`cat`, `cut`, etc.) are currently being enabled by default.  Whereas previously they were available but disabled and had to be activated by one of:

    builtin cat

Or, by adding `/opt/ast/bin` to the $PATH:

    PATH=/opt/ast/bin:$PATH

Or, now by using this variable:

    PATH=${.sh.op_astbin}:$PATH

I have had a quick look at I think this behaviour was accidentally introduced when the `CMDLIST(f)` macro was updated in `src/cmd/ksh93/data/builtins.c` in the Version: 2013-10-10-alpha commit.

I consider this a serious bug which would undoubtedly break many existing scripts not expecting the builtin behaviour.

**Comment by siteshwar** [comment]
I can see one relevant change about libcmd builtins in `RELEASE` file :

```
12-10-04  libcmd builtins are statically linked into ksh93 and by default are
      bound to the path /opt/ast/bin whether this path exists or not.
      Changing the .sh.op_astbin variable changes the binding.
```

There are no references to why these builtins were enabled by default. Since this change will very likely break existing scripts, I propose disabling these builtins by default.

**Comment by kernigh** [comment]
Are these builtins still enabled? I can't find builtin cat.

```
$ type cat
cat is a tracked alias for /bin/cat
$ builtin cat
builtin: cat: not found
$ (PATH=/opt/ast/bin:$PATH; type cat)
cat is a tracked alias for /bin/cat
$ (PATH=/opt/ast/bin:$PATH; builtin cat)
builtin: cat: not found
```

**Comment by krader1961** [comment]
> Are these builtins still enabled? I can't find builtin cat.

Yes and no. Yes, they are presumably still enabled unconditionally since we haven't made any changes to address this issue. No, because when we switched from Nmake to Meson we did not enable building those new builtin commands. That, as is enabling support for dynamically loaded plugins, work to be done to have feature parity with the legacy Nmake based build.
